### DISCUSSION TITLE
Default URL

### DISCUSSION THREAD

**Discussion by virtual-orbis** [YOURLS/YOURLS#3406] [Q&A] [answered]
Hello, 
I was wondering if it is possible to add a default URL for redirection in cases when the visitor does not add anything else to the hostname address (i.e. intranet.mydomain.xx)

**Comment by dgw** [comment, marked as answer]
Solution in https://github.com/YOURLS/YOURLS/discussions/3058#discussioncomment-1310085 should help.
